From d3de4b07a22ceb148e3fb791e4a08d260bf80477 Mon Sep 17 00:00:00 2001
From: ningyu <ningyu@eswincomputing.com>
Date: Thu, 28 Aug 2025 19:08:22 +0800
Subject: [PATCH 503/534] fix:modify pmic's label name of die1

Changelogs:
Chnage pmic's label name of die1 to d1_npu_vdd

Signed-off-by: ningyu <ningyu@eswincomputing.com>
---
 arch/riscv/boot/dts/eswin/eic7702-d560a.dts | 21 ++++++++++++++++-----
 1 file changed, 16 insertions(+), 5 deletions(-)

diff --git a/arch/riscv/boot/dts/eswin/eic7702-d560a.dts b/arch/riscv/boot/dts/eswin/eic7702-d560a.dts
index d189592c2771c..3adfef6dabb17 100644
--- a/arch/riscv/boot/dts/eswin/eic7702-d560a.dts
+++ b/arch/riscv/boot/dts/eswin/eic7702-d560a.dts
@@ -33,22 +33,26 @@ &bootspi {
 
 &sdhci_emmc {
 	/* emmc YXSCG6PE-AK */
+	negedge-data-out;
+};
+
+&d0_i2c5 {
+	/delete-node/ tmp102@48;
 };
 
 &d0_aon_i2c1 {
 	/* temp sensor(gx112) & pmic(ES5340) */
-	/delete-node/ tmp102@48;
 	/delete-node/ tps549d22@10;
 
 	es5340@f {
 		compatible = "einno,es5340";
 		reg = <0xf>;
 		eswin,regulator_default-microvolt=<1000000>;
-		eswin,regulator_label = "supply vdd1", "npu vdd1", "npu current1", "npu temperature1";
+		eswin,regulator_label = "Supply VDD", "D0 NPU Voltage", "D0 NPU CURRENT", "D0 NPU Temperature";
 		label = "npu_vdd";
 		regulators{
 			d0_npu_vcc:npu_svcc{
-				regulator-name="NPU_SVCC_D0";
+				regulator-name="D0_NPU_SVCC";
 				regulator-min-microvolt=<700000>;
 				regulator-max-microvolt=<1100000>;
 				regulator-min-microamp=<20000000>;
@@ -67,11 +71,11 @@ es5340@f {
 		compatible = "einno,es5340";
 		reg = <0xf>;
 		eswin,regulator_default-microvolt=<1000000>;
-		eswin,regulator_label = "supply vdd1", "npu vdd1", "npu current1", "npu temperature1";
+		eswin,regulator_label = "Supply VDD1", "D1 NPU Voltage", "D1 NPU CURRENT", "D1 NPU Temperature";
 		label = "d1_npu_vdd";
 		regulators{
 			d1_npu_vcc:npu_svcc{
-				regulator-name="NPU_SVCC_D1";
+				regulator-name="D1_NPU_SVCC";
 				regulator-min-microvolt=<700000>;
 				regulator-max-microvolt=<1100000>;
 				regulator-min-microamp=<20000000>;
@@ -80,4 +84,11 @@ d1_npu_vcc:npu_svcc{
 			};
 		};
 	};
+};
+
+&npu_opp_table {
+	opp@1500000000 {
+		opp-hz = /bits/ 64 <1500000000>;
+		opp-microvolt = <1080000>;
+	};
 };
\ No newline at end of file
-- 
2.53.0

